.travis.yml

```diff
@@ -3,18 +3,22 @@ language: php
 sudo: false
 
 php:
-  - 5.3
```

Comment:
needed to remove it due to https://docs.travis-ci.com/user/reference/trusty#PHP-images